keps/prod-readiness/sig-instrumentation/647.yaml

```diff
@@ -0,0 +1,3 @@
+kep-number: 647
+alpha:
+  approver: "@wojtek-t"
```

Comment:
Hmm - the PRR doesn't follow the template from the KEP template (which didn't change since May 14th 2020).

Can you please update to the PRR template (which has some questions that aren't answered now).

Reply:
Done!  I only answered required-for-alpha sections.